### DISCUSSION TITLE
Register new Blocks -> how to get SCSS compiled?

### DISCUSSION THREAD

**Discussion by saschapi** [bootscore/bootscore#424] [Q&A]
Hi. I'm working on my first project with bootScore. So far I really like it. 
I'm needing to register some additional Gutenberg blocks and would love to have a dedicated CSS File for it (having those styles in the editor too). 
Is there any option to have an additional SCSS File that would get complied to another CSS File? 

Thanks for any idea.
Cheers Sascha

**Comment by justinkruit** [comment]
Hi Sascha,
Unfortunately, this isn't an option in the current build. The compile script is built to focus only the default files and doesn't have any function to expand this. This may be something we could look into by separating the function into smaller functions so parts can be called to build more files, but this is currently not a direct focus.

**Comment by Axos11** [comment]
Hey, I am not quite sure "what" your plan is, but if you just want to separate your scss files and are fine with it being compiled in the main style afterwards, you could just do an @import path_to_another_scss_file, in the _bscore_custom.scss.
If you do that, keep in mind, that a change on that file WON'T be monitored by the build system. So you would to change "something" in the standard files or be in dev mode for the scss to be recompiled. 

Depending on your Code editor there are also several plugins or built in functions to add a custom "scss -> css" compiliation for files.

And if you want a specific workflow, you still skip the phpscss compilation (thanks to the latest adaptions from justinkruit) and just go your own build path with (gulp, webpack...)

**Reply by justinkruit** [reply]
> If you do that, keep in mind, that a change on that file WON'T be monitored by the build system.

They _should_ actually, as long as they're in the root scss folder. It looks at all the scss and adds up the last modified time of all files.

And for the import part, I get how they're trying to register blocks, and that will need a seperate css file to load that style into the Gutenberg editor. Using imports is fine for the frontend, but won't work in Gutenberg.

**Reply by Axos11** [reply]
you're right. I forgot about that, obv. too long ago i tinkered with blocks myself.

Still, If they are using visual studio code or phpstorm, there won't be an issue to set up the compiling for these blocks in the editor directly.
